Doras/src/app/carousel-tabs/carousel-tabs.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+
+
+@Component({
+  selector: 'doras-carousel-tabs',
+  templateUrl: './carousel-tabs.component.html',
+  styleUrls: ['./carousel-tabs.component.css'],
+})
+export class CarouselTabsComponent implements OnInit {
+
+  constructor() { }
+
+  tabs : string[] = [
```

Comment:
why is this not stored in the DB? tabs is also not a good name! what does the tab represent? capability? if so, call it that!

Reply:
to be completed at a later stage

Reply:
@Thomas-Boyle